Unix/doc/setup-ntlm-omi.md

```diff
@@ -0,0 +1,163 @@
+#Enabling SPNEGO/NTLM Authentication on OMI
+
+OMI requests between the client and server have been up to this point
+authenticated via Basic auth. Basic is very simple, but not at all secure, 
+inasmuch as the username and password are transmitted in the clear, with only 
+Base64 encoding which is easly decoded by someone able to see the traffic on the 
+connection. For that reason we only recommend Basic never be used on other than a 
+secure (https) conneciton.
+
+A more secure method of authentication uses Secure Protected Negotitation protocol, which enables
```

Comment:
You should add `(SPNEGO)` after your spelled out form as a means to define the abbreviation.